src/libstd/build.rs

```diff
@@ -39,8 +39,13 @@ fn main() {
         println!("cargo:rustc-link-lib=framework=Security");
         println!("cargo:rustc-link-lib=framework=Foundation");
         println!("cargo:rustc-link-lib=resolv");
+    } else if target.contains("uwp") {
+        println!("cargo:rustc-link-lib=ws2_32");
+        // For BCryptGenRandom
+        println!("cargo:rustc-link-lib=bcrypt");
     } else if target.contains("windows") {
         println!("cargo:rustc-link-lib=advapi32");
+        println!("cargo:rustc-link-lib=bcrypt");
```

Comment:
Ah sorry if this requires picking up a new dependency on all other Windows platforms, I think we'll want to keep the bcrypt dependency UWP-specific for now.

Reply:
It does, so I'll do the change :)

Reply:
Done